observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-2.adoc

```diff
@@ -0,0 +1,93 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-2"]
+= Release notes for {DTProductName} 3.2
+:context: distributed-tracing-rn-3-2
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+You can use the {DTShortName} xref:../../otel/otel-forwarding.adoc#otel-forwarding-traces[in combination with] the xref:../../otel/otel-installing.adoc#install-otel[{OTELName}].
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+// Tempo section
+[id="distributed-tracing-rn_3-2_tempo-release-notes"]
+== {TempoName}
+
+The {TempoName} is provided through the {TempoOperator}.
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
```

Comment:
This might be added if it gets merged https://github.com/grafana/tempo-operator/pull/910  https://issues.redhat.com/browse/TRACING-4108

Reply:
ACK.